src/gateways/case-notes.test.js

```diff
@@ -0,0 +1,65 @@
+import moxios from 'moxios';
+import { CaseNotesGateway } from './case-notes.js';
+
+describe('Case notes gateway', () => {
+    const csGateway = new CaseNotesGateway();
+
+    beforeEach(() => {
+        moxios.install();
+    });
+
+    afterEach(() => {
+        moxios.uninstall();
+    });
+
+    it('200 Success "/residents/:residendId/caseNotes"', async () => {
```

Comment:
I think, think more carefully about test names, slightly more readable would be nice.

Like Returns status 200 when... - the URL isn't important in the test name

Reply:
Thanks for pointing that out, will update